Source/Core/Core/DSP/DSPAccelerator.h

```diff
@@ -4,18 +4,54 @@
 
 #pragma once
 
-#include <functional>
-
 #include "Common/CommonTypes.h"
 
+class PointerWrap;
+
 namespace DSP
 {
-u16 ReadAccelerator(u32 start_address, u32 end_address, u32* current_address, u16 sample_format,
-                    s16* yn1, s16* yn2, u16* pred_scale, s16* coefs,
-                    std::function<void()> end_exception);
+class Accelerator
```

Comment:
This should probably have a virtual destructor

Reply:
Thanks, fixed both issues.